hybig/browse.py

```diff
@@ -155,6 +155,7 @@ def create_browse_imagery(
     xml file.
 
     """
+    logger.info(message)
```

Comment:
I think this will spit credentials into the log.  I don't know if that's important or not.  I do this for debugging all the time and I think it's super helpful to have the full message available for debugging.  

I have been meaning to send a patch over to the harmony-service-lib to redact that. But  it's not there yet.  

~I don't have strong opinions though, the logs are pretty secure.~ I noticed this was my idea, it should come out probably.